flink-python/dev/lint-python.sh

```diff
@@ -213,11 +213,12 @@ function install_tox() {
         fi
     fi
 
-    # virtualenv 16.6.2 released in 2019-07-14 is incompatible with py27 and py34,
-    # force install an older version(16.0.0) to avoid this problem.
-    $CONDA_PATH install -p $CONDA_HOME -c conda-forge virtualenv=16.0.0 tox -y -q 2>&1 >/dev/null
```

Comment:
What about fix the version for both virtualenv and tox? Then we can avoid the issues introduced by these dependencies?

Reply:
Yes.Fixed version will avoid many issues.

Reply:
+1 to specify the corresponding version.